mgmt/src/models/internal/natconfig/test.rs

```diff
@@ -173,12 +173,30 @@ mod tests {
             remote_id: "67890".try_into().expect("Failed to create VPC ID"),
         };
 
+        // This code is extremely convoluted
+        let mut vpctable = VpcTable::new();
```

Comment:
What does that mean? Any suggestion for improvement in particular?

Reply:
I added this as a reminder that the tests need to be cleaned up so that they are easier to maintain.